### PR TITLE
Fix: Providers no longer burn files randomly

### PR DESCRIPTION
**Pull request by TheMarstonConnell**
Since the dawn of Jackal, providers have randomly burned files. Here is the end to that.

### CODE REVIEW

**Comment by TheMarstonConnell** [comment]
image
These are some metrics showing this off. Burned files stays at zero even after more than 500 files are on the system.

File: x/storage/client/cli/tx_post_file.go [hunk 1]
```diff
@@ -230,12 +230,13 @@ func postFile(fileData []byte, cmd *cobra.Command) {
 		panic(err)
 	}
 
+	if res != nil {
+		fmt.Println(res.RawLog)
```
Comment (dahn510) [inline x/storage/client/cli/tx_post_file.go:234]: is this debugging code?

**Review by dahn510 — APPROVED** [review]
:+1:

File: x/storage/client/cli/tx_post_file.go (hunk 1 above)
Comment (TheMarstonConnell) [reply x/storage/client/cli/tx_post_file.go:234]: Kind of yeah, it's mildly helpful though always

Comment (TheMarstonConnell) [reply x/storage/client/cli/tx_post_file.go:234]: Generally the cli in this instance should really only be for debugging anyways

**Pull request merged**